### ISSUE TITLE
jwt.auth and auth middleware common use

### ISSUE THREAD

**Issue by mediaceh** [tymondesigns/jwt-auth#1085]
Is it possible use it common in controller constructor as 

    public function __construct()
    {
		$this->middleware(['auth','jwt.auth']);
    }
?

**Comment by superbiche** [comment]
Yes. I use it this way (but without the auth middleware that is useless to me).
```public function __construct()  
{  
    $this->middleware('jwt.auth');  
    parent::__construct();  
}  
```
You can even use it partially like explained [in Laravel docs (5.2)](https://laravel.com/docs/5.2/controllers#controller-middleware)  

```
$this->middleware('jwt.auth', ['except' => [
            'index',
            'show'
]]);  
```

[5.3 way](https://laravel.com/docs/5.3/controllers#controller-middleware)  (I'm still using the 5.2 way which is OK)

```
$this->middleware('jwt.auth')->except([
            'index',
            'show'
]);  
```
